ui/public/locales/zh_CN.json

```diff
@@ -3331,7 +3331,7 @@
   "message.enabling.zone": "\u6B63\u5728\u542F\u7528\u8D44\u6E90\u57DF",
   "message.enabling.zone.dots": "\u6B63\u5728\u542F\u7528\u8D44\u6E90\u57DF...",
 
-  "message.enter.seperated.list.multiple.cidrs": "\u5982\u679C\u5B58\u5728\u591A\u4E2A CIDR\uFF0C\u8BF7\u8F93\u5165\u7528\u9017\u53F7\u5206\u9694\u7684 CIDR \u5217\u8868",
+  "message.enter.separated.list.multiple.cidrs": "\u5982\u679C\u5B58\u5728\u591A\u4E2A CIDR\uFF0C\u8BF7\u8F93\u5165\u7528\u9017\u53F7\u5206\u9694\u7684 CIDR \u5217\u8868",
```

Comment:
I do not think this is used anywhere, there is no entry for this in the `en.json` file as well. You could just remove it.